core/models.py

```diff
@@ -280,10 +280,14 @@ def add_organizer(self, email, first_name, last_name):
         return user
 
     def set_random_cover(self):
-        event_picture = StockPicture.objects.random_cover()
-        self.photo = event_picture.photo
-        self.photo_credit = event_picture.photo_credit
-        self.photo_link = event_picture.photo_link
+        try:
+            event_picture = StockPicture.objects.random_cover()
+            self.photo = event_picture.photo
+            self.photo_credit = event_picture.photo_credit
+            self.photo_link = event_picture.photo_link
+        except IndexError:
+            # No StockPicture available
+            pass
```

Comment:
Aside: @amakarudze this looks like the source for the Flikr photos Claire mentioned the other day... 🧐

Reply:
Yes @carltongibson, this has been causing some errors in the past.